figures/mapping-histology-labels.Rmd

```diff
@@ -192,6 +195,9 @@ cancer_group_df <- data.frame(
                                 "EPN",
                                 "Teratoma",
                                 "GNG",
+                                "PAST",
```

Comment:
This might be more appropriately `PA` - https://www.webmd.com/cancer/what-is-pilocytic-astrocytoma

Reply:
I was using this http://oncotree.mskcc.org/#/home (type "pilocytic").

Reply:
From Peter Madsen: 
>I usually call it a JPA but i think that might be a bit outdated. I think PA is more appropriate